selectel/resource_selectel_dbaas_datastore_v1.go

```diff
@@ -250,6 +255,11 @@ func resourceDBaaSDatastoreV1Create(ctx context.Context, d *schema.ResourceData,
 		datastoreCreateOpts.RedisPassword = redisPassword.(string)
 	}
 
+	BackupRetentionDays, DaysIsOk := d.GetOk("backup_retention_days")
```

Comment:
Is it necessary to make `BackupRetentionDays` as package variable ? I suppose it should be defined in local scope.
Also, `ok` value should be named as `ok`, so please rename `DaysIsOk` to ok.